packages/site/src/pages/Article.tsx

```diff
@@ -23,25 +23,27 @@ export const ArticlePage = React.memo(function ({ article }: { article: PageLoad
       references={{ ...article.references, article: article.mdast }}
       frontmatter={article.frontmatter}
     >
-      <ThebeSessionProvider start={false} name={article.slug}>
-        {!hide_title_block && (
-          <FrontmatterBlock kind={article.kind} frontmatter={article.frontmatter} />
-        )}
-        <NotebookProvider siteConfig={false} page={article}>
-          <div className="flex items-center">
-            <div className="flex-grow"></div>
-            {binder && <BinderBadge binder={binder} />}
-            {canCompute && isJupyter && (
-              <EnableCompute canCompute={true} key={article.slug}>
-                <NotebookRunAll />
-              </EnableCompute>
-            )}
-          </div>
-          <ContentBlocks pageKind={article.kind} mdast={article.mdast as GenericParent} />
-          <Bibliography />
-          {!hide_footer_links && <FooterLinksBlock links={article.footer} />}
-        </NotebookProvider>
-      </ThebeSessionProvider>
+      <ThebeRenderMimeRegistryProvider>
+        <ThebeSessionProvider start={false} name={article.slug}>
+          {!hide_title_block && (
+            <FrontmatterBlock kind={article.kind} frontmatter={article.frontmatter} />
+          )}
+          <NotebookProvider siteConfig={false} page={article}>
```

Comment:
I am looking forward to some of these being merged into a single provider as we move to supporting across different articles!